plugins/win-capture/graphics-hook/vulkan-capture.c

```diff
@@ -509,7 +509,9 @@ static void add_surf_data(struct vk_inst_data *idata, VkSurfaceKHR surf, HWND hw
 static HWND find_surf_hwnd(struct vk_inst_data *idata, VkSurfaceKHR surf)
 {
 	struct vk_surf_data *surf_data = (struct vk_surf_data *)get_obj_data(&idata->surfaces, (uint64_t)surf);
-	return surf_data->hwnd;
+	if (surf_data)
+		return surf_data->hwnd;
+	return 0;
```

Comment:
In hindsight, I would have preferred a blank line here to separate it from the `if` contents above.